aspnetcore/signalr/hubs/sample/Modules/HubErrorHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SignalRChat.Modules
+{
+    public class ErrorHandlingPipelineModule : HubPipelineModule
```

Comment:
This doesn't work in SignalR Core. We don't have Pipeline Modules, and there's no equivalent at this time.

Reply:
That was for testing and I forgot to remove it